Seccion 30 (PARTE FINAL) - Proyecto Bienes raíces

CarlosJCD opened the pull request:
Ultima parte de los avances en codigo generados en la seccion 30.

CarlosJCD committed:
Video #325 - Seccion #30 - Index ventana admin

CarlosJCD committed:
Video #326 - Seccion #30 - Inicio de la seccion de actualizacion

CarlosJCD committed:
Video #327 - Seccion #30 - Cargando datos de la propiedad en el form

CarlosJCD committed:
Video #328 - Seccion #30 - Cambiando query para actualizar datos

CarlosJCD committed:
Video #329 - Seccion #30 - Evitar imagenes duplicadas

CarlosJCD committed:
Video #330 - Seccion #30 - Proceso eliminar propiedad

CarlosJCD committed:
Video #331 - Seccion #30 - Seccion anuncios del index dinamica

CarlosJCD committed:
Video #332 - Seccion #30 (FIN SECCION) - Cargando informacion al anuncio

CarlosJCD committed:
Actualizacion archivo base de datos

LUGAMAFE reviewed (changes requested):
Hola carlos te deje comentarios en discord has esos cambios antes de aprobar este PR

CarlosJCD committed:
Fix: Correccion rutas en 'actualizar.php'

CarlosJCD committed:
Fix: Correccion con el charset en la conexion a la bade de datos

CarlosJCD committed:
Imagen prueba

Imagen generada en la prueba de la funcionalidad  de "cambios.php" con respecto a las imagenes

CarlosJCD committed:
Correccion rutas

LUGAMAFE reviewed (approved):
Listo carlos muchas gracias!
